mesonbuild/mformat.py

```diff
@@ -829,7 +829,10 @@ def load_configuration(self, configuration_file: T.Optional[Path]) -> FormatterC
         config = FormatterConfig()
         if configuration_file:
             cp = DefaultConfigParser()
-            cp.read_default(configuration_file)
+            try:
+                cp.read_default(configuration_file)
+            except ParsingError as e:
+                raise MesonException(f'Unable to parse configuration file "{configuration_file}"') from e
```

Comment:
Does this provide a useful description of what kind of error it hit? Maybe we should include the error string from `e` in the MesonException...

Reply:
That's a fair point. I added it.